Neos.Flow/Classes/Http/AbstractMessage.php

```diff
@@ -21,6 +21,7 @@
  *
  * @api
  * @Flow\Proxy(false)
+ * @deprecated
```

Comment:
Please lets always add the version since when this is deprecated and a short comment on why and how to replace it